### PR TITLE
Handel redis caching errors

### PR DESCRIPTION
**Pull request by zhx828**
We will print all errors in log but let it silently skip so website functionality is not affected.

This is related to https://github.com/oncokb/oncokb/issues/3326

### CODE REVIEW

File: src/main/java/org/mskcc/cbio/oncokb/config/cache/CachingConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class CachingConfiguration extends CachingConfigurerSupport {
+    @Override
+    public CacheErrorHandler errorHandler() {
+        return new LoggingCacheErrorHandler();
+    }
+}
+
```
Comment (calvinlu3) [inline src/main/java/org/mskcc/cbio/oncokb/config/cache/CachingConfiguration.java:14]: I'm wondering if its the same if we extend CacheConfiguration (`src/main/java/org/mskcc/cbio/oncokb/config/CacheConfiguration.java`) with `CachingConfigurerSupport` and then register the errorHandler in the same file?

Comment (zhx828) [reply src/main/java/org/mskcc/cbio/oncokb/config/cache/CachingConfiguration.java:14]: That's a good point. Let me try.

Comment (zhx828) [reply src/main/java/org/mskcc/cbio/oncokb/config/cache/CachingConfiguration.java]: @calvinlu3 updated

**Review by calvinlu3 — APPROVED** [review]
LGTM

**Pull request merged**